src/components/sidebar/left-sidebar/TotalStats.tsx

```diff
@@ -2,66 +2,66 @@ import React, { useContext, useEffect, useState } from "react";
 import { AppContext } from "../../../context";
 import { State, PageState } from "../../../types";
 import Translate from "../../../utils/translate/translateService";
-import './TotalStats.css';
+import "./TotalStats.css";

```

Comment:
is it worth to add linting checks to our code and have 1 PR where we fix up linting throughout the app? just finding it hard sometimes to parse out real changes from linting related stuff

Reply:
100% agree with this - we can configure Git hooks and also add a check in our CI / CD